app/models/User.coffee

```diff
@@ -0,0 +1,25 @@
+co       = require 'co'
+mongoose = require 'mongoose'
+
+userSchema = new mongoose.Schema
+	displayName:
+		type: String
+		required: yes
+	id:
+		type: String
+		unique: yes
+		required: yes
+	role:
+		type: String
+		enum: ['game_master', 'player']
+		default: 'player'
+
+userSchema.static 'fromOAuthProfile', co.wrap (profile) ->
```

Comment:
Does it make sense to create a separate layer for it (smth like DAL/service/controller) to manipulate with DB?